doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md

```diff
@@ -0,0 +1,55 @@
+#<a name="top"></a>Grouping Rules
+Grouping rules are an advanced global feature of Cygnus. It is global because it is available for all NGSI sinks.
+
+Such rules are just a Json within a configuration file (that can be instantiated from `grouping_rules.conf.template`) located at `conf/`.
+
+Grouping rules allow changing the notified <b>FIWARE service path</b> and the concatenation of <b>entity ID</b> and <b>entity type</b> (this is called the destination).
+Such rules are just a Json within a configuration file detailing how notified <b>FIWARE service path</b> must be replaced by alternative <b>FIWARE service path</b>.
+
+Grouping rules are writing in Json formal. The following code is just an example:
+
+```
+{
+    "grouping_rules": [
+        {
+            "id": 1,
+            "fields": [
+                ...
+            ],
+            "regex": "...",
+            "destination": "...",
+            "fiware_service_path": "..."
+        },
+        ...
+    ]
+}
+```
+
+When a notification is sent to Cygnus, a special Flume interceptor called `NGSIGroupingInterceptor` intercepts the plain `Event`'s generated by `NGSIRestHandler`, parses the `Event`'s body and replace the <b>FIWARE service path</b> of the `Event` with the notified one in the Grouping Rule. If `fields` it not null the modify is done with the notified fields such as <b>entityId<b>, <b>entityType</b> or </b>servicePath</b>. Once finished, the modified version of the notification is put into the channel buy means of a `NGSIEvent`. Explained fields of a Grouping Rule:
```

Comment:
This must be changed with:

When a notification is sent to Cygnus, a special Flume interceptor called `NGSIGroupingInterceptor` intercepts the plain Flume `Event` generated by `NGSIRestHandler`, parses the `Event`’s body and adds certain new headers (these headers are lists, having one element per notified context element within the notification): the default destinations, the grouped service paths and the grouped destinations. Thus, a sink may choose among default o grouped version of the destination and the grouping. Once finished, the modified event is put again into the channel as a native Flume `Event`.